components/camel-spring-ws/src/main/java/org/apache/camel/component/spring/ws/SpringWebserviceComponent.java

```diff
@@ -75,20 +75,19 @@ private void addConsumerConfiguration(
         if (type != null) {
             LOG.debug("Building Spring Web Services consumer of type {}", type);
             String lookupKey = getLookupKey(remaining, type);
+            configuration.setEndpointMappingType(type);
+            configuration.setEndpointMappingLookupKey(lookupKey);
             if (EndpointMappingType.BEANNAME.equals(type)) {
                 addEndpointDispatcherToConfiguration(configuration, lookupKey);
             } else {
                 addEndpointMappingToConfiguration(parameters, configuration);
+                if (type.equals(EndpointMappingType.XPATHRESULT)) {
```

Comment:
it seem this code is moved into this else block which was not the case before.

Reply:
You are correct, but this block will only execute if the branch where it is now is executed anyway, so I moved them together.

Reply:
A bit of a nitpick, but ideally it should be on a separate commit, as it prevents us from cherry-picking the doc changes to 4.4.x if we would like to.